src/main/java/com/minecolonies/util/EntityUtils.java

```diff
@@ -130,25 +130,6 @@ public static boolean tryMoveLivingToXYZ(EntityLiving living, int x, int y, int
     }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)